modules/tinymce/tools/tasks/emojis.js

```diff
@@ -0,0 +1,48 @@
+const resolve = require('resolve');
+const twemoji = require('twemoji');
+
+const prettyPrint = (obj) => JSON.stringify(obj,null, 2);
+
+const generateContent = (json, attribution) => {
+  const header = 'window.tinymce.Resource.add(\'tinymce.plugins.emoticons\', ';
+  const footer = ');';
+  return '// Source: npm package: emojilib' + attribution + '\n' + header + json + footer;
+};
+
+const getTwemojiOptions = (grunt, options) => {
+  const twemojiOptions = options.twemoji || { ext: '.png' };
```

Comment:
Explicit null checks?

Reply:
This is a build script, so we don't normally have the same code style (see all the other scripts for example). I'm happy to change though, but I'd prefer to keep things consistent.

Reply:
I'm fine with this style for build scrips